proto/iotics/api/twin.proto

```diff
@@ -281,6 +286,9 @@ message UpsertTwinRequest {
     // Feeds with metadata to set to the twin
     repeated UpsertFeedWithMeta feeds = 7;
 
+    // Inputs with metadata to set to the twin
+    repeated UpsertInputWithMeta inputs = 8;
```

Comment:
This contains a Value object, the comment of which relates to feeds and should be updated to include inputs.

Reply:
https://github.com/Iotic-Labs/api/pull/16/commits/9c0cb16f4cdfc87f46bb1de1240239e5e153fc6c